src/server.js

```diff
@@ -314,19 +314,26 @@ const PATH_NAME = process.env.NODE_ENV === "production" ?
   '~/gazelle-production/scripts/restartServers.sh' : process.env.NODE_ENV === "beta" ?
   '~/gazelle-beta/scripts/restartServers.sh' : __dirname + '/scripts/restartServers.sh';
 
+var isrestarted = false;
+
 editorTools.get('/restartserver', (req, res) => {
+
+  isrestarted = true;
+
   if (!process.env.NODE_ENV) {
-    // in dev mode
+    //in dev mode
     res.status(200).send("restarted");
```

Comment:
We should change this dev stuff, maybe send a custom message like `'dev'` or something or if you can think up a good dev solution that would also test the pinging go for it.